README.md

```diff
@@ -348,19 +348,38 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
 
 Once your first push notifications are sent and opened, you should start to see _Opened Push_ metrics within your Klaviyo dashboard.
 
-#### Rich Push
+#### Rich Push (Images & Videos)
 
 >  ℹ️ Rich push notifications are supported in SDK version [2.2.0](https://github.com/klaviyo/klaviyo-swift-sdk/releases/tag/2.2.0) and higher
 
-[Rich Push](https://help.klaviyo.com/hc/en-us/articles/16917302437275) is the ability to add images to push notification messages.  Once the steps
+[Rich Push](https://help.klaviyo.com/hc/en-us/articles/16917302437275) is the ability to add images and videos to push notification messages.  Once the steps
 in the [Installation](#installation) section are complete, you should have a notification service extension in your
 project setup with the code from the `KlaviyoSwiftExtension`. Below are instructions on how to test rich push notifications.
 
+##### Supported Media Formats
```

Comment:
🔍  We should probably make this concise. Seems a little too verbose with sections etc. 

One general theme we try to adhere with the readme is, for a developer who is just looking to get in and do the integration how quickly can they do that.